src/features/communication/communication.tsx

```diff
@@ -0,0 +1,46 @@
+import { Paragraph, Subtitle, Title } from '@/app/components';
+import { DiscordLogo } from '@/icons/discord-logo';
+import {
+  CourseNames,
+  RS_DOCS_COMMUNICATION_LINK,
+  discordLinks,
+} from '@/shared/data/communication.data';
+
+import './communication.scss';
+
+interface RequiredProps {
+  courseName: CourseNames;
+}
+
+export const Communication = ({ courseName }: RequiredProps) => {
+  return (
+    <section className="communication container">
+      <div className="communication content info-wrapper">
+        <Title text="Communication" hasAsterisk />
+        <div className="column-2">
+          <div className="disclogo-wrapper">
+            <DiscordLogo />
+          </div>
```

Comment:
I think this icon should be smaller, currently it is huge

Reply:
Maybe make the icon more colorful? Like app icon?
![Screenshot 2024-05-13 at 17 23 21](https://github.com/rolling-scopes/site/assets/56927722/843d25c7-fbff-45ba-81c3-c41acf16573b)